deeprank/learn/DataSet.py

```diff
@@ -53,6 +54,9 @@ def __init__(self,database, test_database = None,
                 Example : ['1AK4.hdf5','1B7W.hdf5',...]
             test_database (list(str)): names of the hdf5 files used for the test
                 Example : ['7CEI.hdf5']
+            select_pdb (list(str)): names of complexes used for training/validation/test
```

Comment:
I would change that with an option `use_rotation (int) : number of rotations used during the training` 
Otherwise the users would have to provide a very long list of names and that's not great